src/styles/block/cards.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  width: 100%;
+  display: grid;
+  padding-top: 50px;
+  margin: 0 auto;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  place-content: center;
+
+  &__section {
+    width: 200px;
+    padding: 32px 16px 16px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background: $background-color;
+    transition: transform 0.3s ease;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  &__image {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__header {
+    font-family: Roboto, sans-serif;
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    color: $second-color;
+  }
+
+  &__info {
+    margin-top: 24px;
+    display: flex;
+    font-display: row;
+    justify-content: space-between;
+  }
+
+  &__content {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: $second-color;
+  }
+
+  &__raiting {
+    margin-top: 16px;
+    display: flex;
+    font-display: row;
+    justify-content: space-between;
+  }
```

Comment:
Same as above, `font-display: row;` seems to be a typo. Please review this part.